org.civicrm.hremergency/info.xml

```diff
@@ -2,19 +2,19 @@
 <extension key="org.civicrm.hremergency" type="module">
   <file>hremergency</file>
   <name>Emergency Contacts (CiviHR)</name>
-  <description></description>
+  <description>Emergency Contacts extensions for CiviHR</description>
```

Comment:
I think this should be `<description>Emergency Contacts extension for CiviHR</description>`.
`extensions` => `extension`